bin/sngl/pycbc_export_censored_trigs

```diff
@@ -102,8 +105,39 @@ else:
     trigs_censored = ForegroundTriggers(opts.coinc_file, opts.bank_file,
                           sngl_files=opts.sngl_files, group=group)
 
-# find triggers above limit
-mask = numpy.where( trigs_censored.get_coincfile_array("stat") > opts.min_stat )[0]
+# apply clustering to censored_triggers if a cluster-window has been parsed in the command line else skip clustering. Then pick triggers above min-stat if one is parsed in the command line
+
+# The clustering picks the loudest trigger in each window
+
+if opts.cluster_window:
+    coinc_newsnr = numpy.array( trigs_censored.get_coincfile_array("stat") )
+
+    time1 = numpy.array( trigs_censored.get_coincfile_array("time1") )
+
+    t_start = numpy.amin(time1)
+    t_end = numpy.amax(time1)
+
+    nbins = (t_end - t_start)/opts.cluster_window
+
+    i = 1
+    t_low = t_start
+    cluster_idx = []
+    while i < (nbins + 1):
```

Comment:
An alternative to this ``while`` loop would be using ``for i in range(n)`` (note the first index will be zero).

Also ``for i in range(n)`` should be faster than ``while i < n``.